projects/driver_controls/src/power_fsm.c

```diff
@@ -39,9 +50,20 @@ static void prv_state_off(FSM *fsm, const Event *e, void *context) {
   EventArbiterCheck *event_check = fsm->context;
   *event_check = prv_check_off;
 
+  PowerFSMState power_state = POWER_FSM_STATE_OFF;
+
+  State *current_state = fsm->current_state;
+
+  if (current_state == &state_off_brake) {
```

Comment:
This logic is a little confusing because you're essentially doing
```
if current_state == state_off:
  output_can(state_off)
else if current_state == state_charging:
  output_can(state_charged)
else if current_state == off:
  // do nothing
```
but through a missing else case and return. :P

What is `POWER_FSM_STATE_CHARGED` supposed to represent? Shouldn't it be charging, not charged?
We're definitely going to want some sort of indicator representing whether we're charging or active @ckitagawa @taipingli

Reply:
The state indicator will be on driver display? Although we can add a light if you want.

Reply:
Driver display won't boot in time to know what state it's in unless it asks.

Reply:
@Derpalooza